odoo/models.py

```diff
@@ -4241,13 +4232,12 @@ def _create(self, data_list):
             # batch, to avoid regressions as much as possible.
             #
             # That said, we haven't closed the door completely.
-            query = "INSERT INTO {} ({}) VALUES ({}) RETURNING id".format(
-                quote(self._table),
-                ", ".join(quote(name) for name, fmt, val in columns),
-                ", ".join(fmt for name, fmt, val in columns),
-            )
-            params = [val for name, fmt, val in columns]
-            cr.execute(query, params)
+            if not columns:  # Manage the case where we create empty record
+                cr.execute(f'INSERT INTO "{self._table}" DEFAULT VALUES RETURNING id')
```

Comment:
An alternative would be to use column `id` with a default value:
```suggestion
            if not columns:  # Manage the case where we create empty record
                columns = [('id', AsIs('DEFAULT'))]
```

Reply:
Yes, it will be the solution for the bulk insert (because `INSERT INTO "{self._table}" DEFAULT VALUES RETURNING id` create only one row). But for now, I prefer to avoid useless parsing, it is ok for you?

Reply:
What do you mean by "useless parsing" ?  My suggestion simply was to turn the special case back to a normal case.

Reply:
Sorry, my message was unclear: by "useless parsing", I mean ask to psycopg to mogrify params into the query for nothing (when the query can be static).
But ok for the change, because it will be the trick for the bulk insert anyway.